contributors/design-proposals/node-allocatable.md

```diff
@@ -89,61 +73,217 @@ The flag will be specified as a serialized `ResourceList`, with resources define
 --kube-reserved=cpu=500m,memory=5Mi
 ```
 
-Initially we will only support CPU and memory, but will eventually support more resources. See
-[#16889](https://github.com/kubernetes/kubernetes/pull/16889) for disk accounting.
-
-If KubeReserved is not set it defaults to a sane value (TBD) calculated from machine capacity. If it
-is explicitly set to 0 (along with `SystemReserved`), then `Allocatable == Capacity`, and the system
-behavior is equivalent to the 1.1 behavior with scheduling based on Capacity.
+Initially we will only support CPU and memory, but will eventually support more resources like [local storage](#phase-3) and io proportional weights to improve node reliability.
 
 #### System-Reserved
 
 In the initial implementation, `SystemReserved` will be functionally equivalent to
 [`KubeReserved`](#kube-reserved), but with a different semantic meaning. While KubeReserved
 designates resources set aside for kubernetes components, SystemReserved designates resources set
 aside for non-kubernetes components (currently this is reported as all the processes lumped
-together in the `/system` raw container).
+together in the `/system` raw container on non-systemd nodes).
+
+## Kubelet Evictions Tresholds
+
+To improve the reliability of nodes, kubelet evicts pods whenever the node runs out of memory or local storage.
+Together, evictions and node allocatable help improve node stability.
+
+As of v1.5, evictions are based on `Capacity` (overall node usage). Kubelet evicts pods based on QoS and user configured eviction thresholds.
```

Comment:
Not sure what the "(overall node usage)" is trying to clarify.  
Possibly rephrase: evictions are based on overall node usage relative to `Capacity`